### DISCUSSION TITLE
AI driven vote result analyse

### DISCUSSION THREAD

**Discussion by ManAnRuck** [demokratie-live/democracy-client#1590] [Ideas]
[Raw Data](https://gist.github.com/ManAnRuck/c430a8574514b5f982245689c732ba2c)
[Votes Toggle logic](https://github.com/demokratie-live/democracy-development/blob/master/services/cron-jobs/sync-procedures/src/index.ts#L157-L191)

**Comment by visionmakery** [comment]
## Prosatexttypen


### 1) Abstimmung über Gesetzesentwurf / Antrag

#### Fraktionen als „Eigenname-Fraktion“
Ich bitte diejenigen, die dem Gesetzentwurf zustimmen wollen, um das Handzeichen. – **CDU/CSU- und AfD-Fraktion**. Wer stimmt dagegen? – Die Fraktionen von SPD, Bündnis 90/Die Grünen, FDP und Die Linke. Damit ist der Gesetzentwurf in zweiter Beratung abgelehnt. Damit entfällt nach unserer Geschäftsordnung die weitere Beratung.

#### „Rest des Hauses“
Wer stimmt für diesen Antrag? – Das ist die AfD. Wer stimmt dagegen? – Das ist der **Rest des Hauses**. Enthaltungen? – Keine. Der Antrag ist damit abgelehnt.

#### „übrigen Fraktionen des Hauses“
Wer stimmt für diesen Antrag? – Die Fraktion Die Linke. Wer stimmt dagegen? – Das sind die **übrigen Fraktionen des Hauses**. Damit ist der Antrag abgelehnt. Enthaltungen, Herr Baumann, gab es keine.

#### „Oppositionsfraktionen“
Wer stimmt für diesen Antrag? – Wer stimmt dagegen? – Wer enthält sich? – Niemand. Der Antrag ist mit den Stimmen der Koalitionsfraktionen gegen die Stimmen der Oppositionsfraktionen angenommen.

#### „Regierungskoalition“
und Schlussabstimmung. Ich bitte diejenigen, die dem Gesetzentwurf zustimmen wollen, sich zu erheben. – Das sind Die Linke, CDU/CSU, die **Regierungskoalition**. Wer stimmt dagegen? – Wer enthält sich? – Das ist die Fraktion der AfD. Der Gesetzentwurf ist damit angenommen.

### 2) Abstimmung über Beschlussempfehlung

Fall 01: Der Ausschuss empfiehlt in seiner Beschlussempfehlung die Ablehnung des Antrags/Gesetzes
Fall 02: Der Ausschuss empfiehlt in seiner Beschlussempfehlung die Annahme des Antrags/Gesetzes

Im ersteren Fall 01 ist dann eine Zustimmung zur Beschlussempfehlung eine Ablehnung des Antrags, d.h. invers zu zählen
Im zweiten Fall 02 ist die Zustimmung zur Beschlussempfehlung auch eine Zustimmung zum Antrag, d.h. gleichlautend.


#### Fraktionen als „Eigennamen“

Schließlich empfiehlt der Ausschuss unter Buchstabe d die Ablehnung des Antrags der Fraktion Die Linke auf Drucksache 20/6546 mit dem Titel „Gute Pflege stabil finanzieren“. Wer stimmt für die Beschlussempfehlung? – Das sind die **SPD, Bündnis 90/Die Grünen, FDP, CDU/CSU und AfD**. Wer ist dagegen? – Das ist die Fraktion Die Linke. Enthaltungen? – Sehe ich nicht. Die Beschlussempfehlung ist damit angenommen.

#### Fraktionen als „Eigennamefraktion“
Der Ausschuss empfiehlt in seiner Beschlussempfehlung auf Drucksache 20/3501, den Antrag der Unionsfraktion auf Drucksache 20/3489 abzulehnen. Wer ist für diese Beschlussempfehlung? – Die Koalitionsfraktionen und Die Linke. Wer stimmt dagegen? – Die **Unionsfraktion**. Wer enthält sich? – Das ist die AfD. Die Beschlussempfehlung ist damit angenommen.

#### „Ampelfraktionen“

Der Ausschuss für Gesundheit empfiehlt unter Buchstabe c seiner Beschlussempfehlung die Ablehnung des Antrags der Fraktion der AfD auf Drucksache 20/4669 mit dem Titel „Pflegeversicherung – Bürokratie abbauen, Pflegebedürftige und ihre Angehörigen entlasten“. Wer stimmt für diese Beschlussempfehlung? – Das sind die **Ampelfraktionen**, die CDU/CSU-Fraktion und auch die Fraktion Die Linke. Wer stimmt dagegen? – Die AfD-Fraktion. Enthaltungen? – Kann ich nicht erkennen. Die Beschlussempfehlung ist damit angenommen.

#### „Koalitionsfraktionen“

Der Ausschuss empfiehlt in seiner Beschlussempfehlung auf der Drucksache 20/6757, den Antrag der Fraktion der CDU/CSU auf Drucksache 20/3948 abzulehnen. Wer stimmt für diese Beschlussempfehlung? – Das sind Die Linke und die **Koalitionsfraktionen**. Wer stimmt dagegen? – CDU/CSU. Wer enthält sich? – Die AfD-Fraktion. Die Beschlussempfehlung ist damit angenommen.

#### „regierungstragenden Fraktionen“

Der Ausschuss empfiehlt in seiner Beschlussempfehlung auf Drucksache 20/6576, den Antrag der Fraktion der CDU/CSU auf Drucksache 20/4315 abzulehnen. Wer stimmt für diese Beschlussempfehlung? – Die **regierungstragenden Fraktionen**, die Fraktion der AfD und Die Linke. Wer stimmt dagegen? – Die CDU/ CSU-Fraktion. Enthaltungen? – Keine. Damit ist die Beschlussempfehlung angenommen.

#### „Opposition“
Der Ausschuss empfiehlt in seiner Beschlussempfehlung auf Drucksache 19/3800, den Antrag der Fraktion der FDP auf Drucksache 19/1839 abzulehnen. Wer stimmt für diese Beschlussempfehlung? – Die Koalitionsfraktionen. Wer stimmt dagegen? – Die Opposition. Wer enthält sich? – Die Beschlussempfehlung ist mit den Stimmen der Koalitionsfraktionen gegen die Stimmen der AfD-Fraktion, der FDP-Fraktion, der Fraktion Die Linke und der Fraktion Bündnis 90/Die Grünen angenommen.

**Comment by appinteractive** [comment]
@ManAnRuck @visionmakery hier hab ich einen Fall aus eurem Export wo es für mich als Laien so aussieht als wäre GPT richtig und ihr falsch. Oder übersehe ich da was?

`expected` ist aus dem Export, `actual` von der API.

```json
{
    "text": "Wer stimmt für die Beschlussempfehlung? – Das sind die Grünen, FDP, Die Linke, CDU/CSU und die SPD . – Gegenprobe! – AfD . Enthaltungen? – Keine . Damit ist die Beschlussempfehlung angenommen .",
    "isValid": false,
    "votingDocument": {
      "valid": false,
      "expected": "mainDocument",
      "actual": "recommendedDecision"
    },
    "validateVotes": {
      "Union": {
        "valid": false,
        "expect": "NO",
        "actual": "YES"
      },
      "SPD": {
        "valid": false,
        "expect": "NO",
        "actual": "YES"
      },
      "AfD": {
        "valid": false,
        "expect": "YES",
        "actual": "NO"
      },
      "Grüne": {
        "valid": false,
        "expect": "NO",
        "actual": "YES"
      },
      "FDP": {
        "valid": false,
        "expect": "NO",
        "actual": "YES"
      },
      "Linke": {
        "valid": false,
        "expect": "NO",
        "actual": "YES"
      }
    }
  },
  ```
